jsx-runtime/src/index.js

```diff
@@ -53,6 +53,10 @@ function createVNode(type, props, key, isStaticChildren, __source, __self) {
 		_hydrating: null,
 		constructor: undefined,
 		_original: --vnodeId,
+		_prevVNode: undefined,
+		_insert: false,
+		_matched: false,
+		_index: -1,
```

Comment:
There's definitely opportunity to consolidate these, and possibly remove obsolete fields once we further take advantage of this new approach. Will explore that more once I get Suspense tests working.